xclim/indicators/atmos/_conversion.py

```diff
@@ -215,3 +217,16 @@ class Converter(Indicator):
     ),
     compute=wrapped_partial(indices.wind_chill_index, mask_invalid=True),
 )
+
+
+potential_evapotranspiration = Converter(
+    identifier="potential_evapotranspiration",
+    units="mm day-1",
+    standard_name="water_evapotranspiration_flux",
+    long_name="Potential evapotranspiration",
```

Comment:
Add `var_name` with the standard CMIP6 var name (by default `identifier` is used).